testing/citest/tests/google_front50_test.py

```diff
@@ -64,7 +64,8 @@ def _do_init_bindings(self):
     context = ExecutionContext()
     config = self.agent.deployed_config
     enabled = config.get('spinnaker.gcs.enabled', False)
-    if not enabled:
+    # We don't read the config files while using 'native' platform.
+    if self.bindings['HOST_PLATFORM'] != 'native' and not enabled:
```

Comment:
Is there another way to do this? I want to get rid of HOST_PLATFORM. At least I think I do. I havent found a need until now.
Instead could you change 66 to default to None and make this a check against False?